### DISCUSSION TITLE
Texture compression

### DISCUSSION THREAD

**Discussion by SSD-aimedis** [donmccurdy/glTF-Transform#717] [Q&A] [answered]
Hi, can anyone please suggest a method for gpu texture compression through script(using functions) for models. I can achieve it by CLI implementation by invoking the specific command but struggling to do the same through node server.
ref : https://gltf-transform.donmccurdy.com/functions.html

**Comment by donmccurdy** [comment, marked as answer]
There's discussion in the issue here:

- https://github.com/donmccurdy/glTF-Transform/issues/675

For now I don't have a solution for you, but the ideas in the thread may help if you'd like to implement such a function yourself, without needing a CLI tool.

**Comment by SSD-aimedis** [comment]
Thank you for the clarification Don McCurdy. This thread really did help me to look into the right way to approach the issue. Will also discuss the same with you if I hopefully get a solution for the same.
